### ISSUE TITLE
datetime-malformed-no-z.toml is not malformed according to the TOML project

### ISSUE THREAD

**Issue by joelself** [BurntSushi/toml-test#35]
From the [TOML project home page](https://github.com/toml-lang/toml#datetime):

> You may omit the local offset and let the parser or host application decide that information. A good default is to use the host machine's local offset.
> 
> ``` toml
> 1979-05-27T07:32:00
> 1979-05-27T00:32:00.999999
> ```

The date time in `datetime-malformed-no-z.toml` is `1987-07-05T17:45:00` which is valid.

**Comment by robmuh** [comment]
Could this be one of many things that changed between v0.2.0 and v0.4.0 (the now official version)?

**Comment by joelself** [comment]
Looking at the README.md of the 2.0 tag of TOML it looks like you're right:

```md
Datetimes are ISO 8601 dates, but only the full zulu form is allowed.

1979-05-27T07:32:00Z
```

**Comment by uiri** [comment]
Hi Joel, thanks for opening this issue. I am closing it as fixed by #41
